crates/sui-graphql-e2e-tests/tests/event_connection/event_connection.move

```diff
@@ -100,10 +100,10 @@ module Test::M2 {
 
 //# create-checkpoint
 
-//# run-graphql --variables A
+//# run-graphql
 {
   eventConnection(
-    filter: {sender: $A}
+    filter: {sender: "@{A}"}
```

Comment:
Sorry to be annoying, but could you leave the other variable uses untouched? This will conflict with the stack dealing with interpolation -- just make sure the one that you added was using interpolation instead of a variable.

Reply:
I see, in that case I haven't added any tests, just modified existing ones, so I'll put the variable uses back